.github/workflows/tools.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: User space tools/
```

Comment:
trailing slash unneeded

Reply:
(too) terse allusion to the `tools/` subdirectory. How about: `User space in tools/`?